packages/components/document-card/src/document-card.js

```diff
@@ -0,0 +1,48 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './document-card.css';
+import fileIcon from './assets/document.svg';
+
+customElementDefineHelper(
+	'ts-document-card',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				name: { type: String, reflect: true },
```

Comment:
Consider to add the `dir` property to support Right-To-Left. Also add a fallback to a body `dir`. You can use the `header` component as an example.

Reply:
ok